101-recovery-services-backup-iaasvm/azuredeploy.json

```diff
@@ -0,0 +1,50 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "existingRecoveryServicesVaultName": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of the Existing Recovery Services Vault"
+            }
+        },
+        "existingProtectionContainers": {
+            "type": "array",
+            "metadata": {
+                "description": "Array of iaasvm protection containers. This will take different input for classic and ARM vms. e.g. iaasvmcontainer;iaasvmcontainerv2;my-resource-group;my-arm-vm for ARM vm and iaasvmcontainer;iaasvmcontainer;my-classic-vm;my-classic-vm for classic vm"
+            }
+        },
+        "existingProtectableItems": {
+            "type": "array",
+            "metadata": {
+                "description": "Array of iaasvm protected items. This will take different input for classic and ARM vms. e.g. vm;iaasvmcontainerv2;my-resource-group;my-arm-vm for ARM vm and vm;iaasvmcontainer;my-classic-vm;my-classic-vm for classic vm"
+            }
+        },
+        "existingBackupPolicyName": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of existing Backup Policy in same Recovery Services Vault."
+            }
+        }
+    },
+    "variables": {
+        "backupFabric": "Azure",
+        "protectedItemType": "Microsoft.ClassicCompute/virtualMachines"
```

Comment:
No it works for both ARM and classic. RP identifies this based on existingProtectionContainers and existingProtectableItems value